back/app/models/user.rb

```diff
@@ -133,11 +135,15 @@ def from_token_request(request)
   before_validation :generate_slug
   before_validation :sanitize_bio_multiloc, if: :bio_multiloc
   before_validation :assign_email_or_phone, if: :email_changed?
+  with_options if: -> { user_confirmation_enabled? } do
+    with_options if: :email_changed?, on: :create do
```

Comment:
Can you nest `with_options`? I think at some point we had a bug somewhere because one of the `with_options` call would undo the other `with_options` call.

Reply:
I don't think Rails can be smart enough to combine both ifs, so I think something is wrong there. But if it was already like that then fine for me.

Reply:
Have changed the way it is called to simplify slightly